_includes/tutorials/udf/ksql/markup/dev/create-inputs.adoc

```diff
@@ -0,0 +1,5 @@
+First, you'll need to create a Kafka topic and stream to represent the stock quote stream. The following creates both in one shot:
```

Comment:
This section has "Firstly" above, too. Probably simpler to leave all the ordinal intros out?
```suggestion
You'll need to create a Kafka topic and stream to represent the stock quote stream. The following creates both in one shot:
```